test/e2e_node/memory_eviction_test.go

```diff
@@ -167,6 +109,67 @@ var _ = framework.KubeDescribe("MemoryEviction [Slow] [Serial] [Disruptive]", fu
 				},
 			})
 		})
+
+		It("should evict pods in the correct order (besteffort first, then burstable, then guaranteed)", func() {
```

Comment:
Why are you moving this section to AfterEach? Isn't this the main purpose of the test?